src/Steam.Common/App.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+
+namespace Steam
+{
+    /// <summary>
+    /// Main representation of a product, see https://partner.steamgames.com/doc/store/application
+    /// </summary>
+    public readonly struct App : IEquatable<App>
+    {
+        /// <summary>
+        /// The Id
+        /// </summary>
+        public readonly uint Id;
```

Comment:
Thinking about how these structures will be used, I think it's a good idea to call the internal values "Value" since it's generic and the struct doesn't represent an App, it represents the ID. On top of this, in other future libraries, properties will expose IDs through this struct, so accessing the raw value manually via something like `Id.Id` would be a bit redundant.

Reply:
This review applies to the other wrappers as well.